sgv2-service-common/src/main/java/io/stargate/sgv2/common/grpc/DefaultStargateBridgeClient.java

```diff
@@ -130,7 +130,9 @@ public CompletionStage<Optional<CqlKeyspaceDescribe>> getKeyspaceAsync(
   private CompletionStage<CqlKeyspaceDescribe> getAuthorizedKeyspace(String keyspaceName) {
     CompletableFuture<CqlKeyspaceDescribe> result = new CompletableFuture<>();
 
-    CqlKeyspaceDescribe cached = keyspaceCache.getIfPresent(keyspaceName);
+    String decoratedKeyspaceName = decorateKeyspaceName(keyspaceName);
```

Comment:
Not sure where it'd make most sense, but I think a note on why decorated keyspace name is needed locally vs not sent via bridge (since bridge side decorates it?) would be useful.
And maybe `fetchKeyspaceFromBridge()` could take `undecoratedKeyspaceName` or something to make this clear(er)?